memgpt/embeddings.py

```diff
@@ -38,6 +37,7 @@ def __init__(
         self._user = user
         self._base_url = base_url
         self._timeout = timeout
+        self._encoding = tiktoken.get_encoding(model)
```

Comment:
I think to get this to pass tests we may just need to wrap this:

```python
try:
    self._encoding = tiktoken.get_encoding(model)
except:
    self._encoding = tiktoken.get_encoding('text-embedding-ada-002')
```

Tests log:
```
FAILED tests/test_client.py::test_create_agent - ValueError: Unknown encoding BAAI/bge-large-en-v1.5. Plugins found: ['tiktoken_ext.openai_public']
FAILED tests/test_client.py::test_user_message - AttributeError: 'NoneType' object has no attribute 'id'
FAILED tests/test_client.py::test_save_load - AssertionError: Run create_agent test first
assert None is not None
FAILED tests/test_server.py::test_server - ValueError: Unknown encoding BAAI/bge-large-en-v1.5. Plugins found: ['tiktoken_ext.openai_public']
```

Reply:
@cpacker I'm uncertain as to whether it is good to that particular fallback unless it is for CI testing only. The problem is that counting tokens with one model before the insert and then actually doing the insert with another model may lead to strange behavior. But I expect you understand that at least as well as I do. So please LMK if you still think your suggested fix is still the right direction (for now) or whether I should try to investigate a little deeper before we merge.

I know you are trying to cut the next release as soon as tomorrow. Is this something you really want in that release? Perhaps that means I should just do your suggested change and then we can follow up with another issue.

If you do want the change ASAP then I will take a few moments from my day job and make that change by say 5pm Pacific Time (I am just across the bay from you in SF).

Reply:
Nice find!

For the meantime, let's actually just call the `utils.count_tokens` method and let it default to `gpt-4` just to get something runnable: https://github.com/cpacker/MemGPT/blob/main/memgpt/local_llm/utils.py#L43-L45

In a follow-up PR we can modify the util method to support an offline mode that uses an alternate tokenizer (that seems more sane than caching the file + including it in the repo + copying it over via shutil to the site-packages location, but idk).

Reply:
Whoops missed the diff - that also works, though maybe we want to still pack this into utils and have the try/except happen there.

But this still LGTM, no need for this PR unless you didn't see the util function and would prefer to move it there.

Reply:
Ok I can create a follow-on PR soon to tidy this up as you propose. In the meantime, since you have verified that the `--extensions` problem I am seeing is just something with my setup then I think we should merge.